lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT_PATH="$DIRNAME/../../.."
+cd $LH_ROOT_PATH
+
+TAR_URL="https://drive.google.com/a/chromium.org/uc?id=1_w2g6fQVLgHI62FApsyUDejZyHNXMLm0&amp;export=download"
```

Comment:
plz add comment on what this is and if it's ever updated frozen snapshots from XXX date